game/operation/operation.py

```diff
@@ -76,6 +75,7 @@ def __init__(self,
         self.to_cp = to_cp
         self.is_quick = False
         self.plugin_scripts: List[str] = []
+        self.subscribed_generators = [KneeboardGenerator, BriefingGenerator]
```

Comment:
If possible, don't add any members to this type. This ends up serialized as part of the `Game` so this affects save game compat. It looks like these could just as easily be passed when `notife_subscribed_generators` is called.

(the fact that this gets serialized is probably a bug in its own right, fwiw, but it looks like a fair amount to untangle)

Reply:
Thoughts on extending the pydcs Mission class and wrapping the save method to notify these mission info classes?

Both the kneeboards and the briefing were being generated twice previously, and it looks like the second generation was to fix a bug where the first was occurring.  

Any time we save a mission file, we want to create the briefing and the kneeboards, and we know the required data is guaranteed to be in it's final state.

Reply:
removed the properties that were added.  notify_subscribed_generators (renamed notify_info_generators in 18a93ef) was also added to the Operator class, I imagine the new method won't cause the save compat problems?

Reply:
> Thoughts on extending the pydcs Mission class and wrapping the save method to notify these mission info classes?

Sounds good. Fine to do as a follow up if you prefer.